File: Java/#1491_average.java
class Solution {
    public double average(int[] salary) {
        int min=Integer.MAX_VALUE;
        int max=-1;
        double sum=0;
        int count=0;
        for(int i=0;i<salary.length;i++){
            max = Math.max(salary[i],max);
            min=Math.min(salary[i],min);
        }
        for(int i=0;i<salary.length;i++){
            if(salary[i]!=min&&salary[i]!=max){
                sum+=salary[i];
                count++;
            }
        }
        double avg = sum/count;
        return avg;
        
    }
}

File: Python/#1491_average.py
class Solution:
    def average(self, salary: List[int]) -> float:
        mins = min(salary)
        maxs=max(salary)
        avg=0
        count=0
        for i in salary:
            if i!=mins and i!=maxs:
                avg+=i
                count+=1
        return avg/count
